docs/index.rst

```diff
@@ -86,126 +80,70 @@ Run demo
-   ``connect-elasticsearch-ksql`` and ``connect-replicator``. These
-   consumer groups will be in the consumer group statistics in the
-   `stream
-   monitoring <https://docs.confluent.io/current/control-center/docs/monitoring.html>`__
-   charts.
-
-   .. figure:: images/connect_consumer_group.png
-      :alt: image
-
-
-4. **MONITORING –> System Health**: to identify bottlenecks, you can
-   see a breakdown of produce and fetch latencies through the entire
-   `request
-   lifecycle <https://docs.confluent.io/current/control-center/docs/systemhealth.html>`__.
-   Click on the line graph in the ``Request latency`` chart. The request
-   latency values can be shown at the median, 95th, 99th, or 99.9th
-   percentile. Depending on where the bottlenecks are, you can tune your
-   brokers and clients appropriately.
-
-   .. figure:: images/request_latencies.png
+   .. figure:: images/topic_actions.png
       :alt: image
 
-
-Topic Management
---------------------------------
-
-|c3| has a useful interface to manage topics in a Kafka cluster.
-
-1. **MANAGEMENT –> Topics**: Scroll down to and click on the topic `wikipedia.parsed`.
-
-      .. figure:: images/topic_actions.png
-         :alt: image
-
-2. **MANAGEMENT –> Topics -> Status**: View which brokers are leaders for which partitions and where all partitions reside.
+4. View which brokers are leaders for which partitions and where all partitions reside.
 
    .. figure:: images/topic_info.png
       :alt: image
 
-3. **MANAGEMENT –> Topics -> Schema**: View the schema for this topic. For `wikipedia.parsed`, the topic value is using a Schema registered with |sr| (the topic key is just a string).
+5. Inspect messages for this topic, in real-time.
 
-   .. figure:: images/topic_schema.png
+   .. figure:: images/topic_inspect.png
       :alt: image
 
-4. **MANAGEMENT –> Topics -> Inspect**: View messages for this topic, in real-time.
+6. View the schema for this topic. For `wikipedia.parsed`, the topic value is using a Schema registered with |sr| (the topic key is just a string).
 
-   .. figure:: images/topic_inspect.png
+   .. figure:: images/topic_schema.png
       :alt: image
 
-5. **MANAGEMENT –> Topics -> Settings**: View configuration settings for this topic.
+7. View configuration settings for this topic.
 
    .. figure:: images/topic_settings.png
       :alt: image
 
-6. **MANAGEMENT -> Topics**: click the ``+ Create`` button on the top right to create a new topic in your Kafka cluster. You can also view and edit settings of Kafka topics in the cluster. Read more on |c3| `topic management <https://docs.confluent.io/current/control-center/docs/topics.html>`__.
+8. Click the ``+ Add a topic`` button on the top right to create a new topic in your Kafka cluster. You can also view and edit settings of Kafka topics in the cluster. Read more on |c3| `topic management <https://docs.confluent.io/current/control-center/docs/topics.html>`__.
```

Comment:
User might need a quick direction to return to the all topics view to see the Add a topic button.

```suggestion
8. Return to the ``All topics`` view and click the ``+ Add a topic`` button on the top right to create a new topic in your Kafka cluster. You can also view and edit settings of Kafka topics in the cluster. Read more on |c3| `topic management <https://docs.confluent.io/current/control-center/docs/topics.html>`__.
```

Reply:
Ack